app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+
+  belongs_to :author, class_name: 'User'
+
```

Comment:
- [ ] <b>OPTIONAL:</b> Hint: It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private